bokehjs/src/coffee/models/tools/gestures/select_tool.coffee

```diff
@@ -14,7 +15,7 @@ export class SelectToolView extends GestureToolView
 
       if renderers.length == 0
         all_renderers = @plot_model.plot.renderers
-        renderers = (r for r in all_renderers when r instanceof GlyphRenderer)
+        renderers = (r for r in all_renderers when r instanceof GlyphRenderer or r instanceof GraphRenderer)
```

Comment:
I don't believe so because TileRenderer doesn't have a `hit_test` method. We could add it and have it do nothing (or implement some sort of tile hit testing), but that's probably another PR.